app/models/episode_story_handler.rb

```diff
@@ -93,12 +94,15 @@ def update_audio
   end
 
   def update_image
-    image_url = begin
-      story.objects['prx:image'].links['original'].href
-    rescue StandardError
-      nil
+    cms_image = story.objects['prx:image'] rescue nil
+    cms_href = cms_image.links['original'].href rescue nil
+
+    episode.image_file = cms_href
```

Comment:
NOTE: if the CMS Story _had no image_ we still need to set this to nil.  Because that triggers the `images.destroy_all` functionality in the setter.

But, only sync the caption/credit if we still have an `image_file` when all is said and done.